src/db/inout/exportcsv.cc

```diff
@@ -138,7 +138,7 @@ void
 {
     // 0.) tntdb connection
     tntdb::Connection conn;
-    std::string msg{"no connection to database"};
+    std::string msg{TRANSLATE_ME ("no connection to database")};
```

Comment:
Do you think we should add #include <fty_common_macros.h> to make it more clear there are macros used?